webpack.dev.js

```diff
@@ -1,8 +1,16 @@
 const path = require("path");
+const fs = require("fs");
+const dotenv = require("dotenv");
 const { merge } = require("webpack-merge");
 const common = require("./webpack.common.js");
 
-const sdkEnv = process.env.sdkEnv;
+// * Load environment variables from .env file
+const envPath = path.resolve(__dirname, ".env");
+const envVars = fs.existsSync(envPath)
+  ? dotenv.parse(fs.readFileSync(envPath))
+  : {};
+
+const sdkEnv = envVars.SDK_ENV ?? "local";
```

Comment:
Shouldn't the usage be 

```js
require('dotenv').config()
``` 
as mentioned [here](https://www.npmjs.com/package/dotenv)?

Reply:
Both works same almost `dotenv.config()` and `dotenv.parse()`
I'm parsing the file same as 
```
// I'm setting up the process while config automatically does for us.
const envVars = fs.existsSync(envPath)
  ? dotenv.parse(fs.readFileSync(envPath))
  : {};
```

But yes it can be changed. Updated.